fpx/src/api.rs

```diff
@@ -74,6 +75,7 @@ impl Builder {
             )
             .with_state(api_state)
             .fallback(StatusCode::NOT_FOUND)
+            .layer(CorsLayer::new().allow_origin(Any))
```

Comment:
We should make this a opt-in, since it will be used by fpx-cli and fpx-workers as well. So just enable it for fpx-app for now.